config/samples/v2/dspa-simple/dspa_simple.yaml

```diff
@@ -7,6 +7,9 @@ spec:
   apiServer:
     enableSamplePipeline: true
   objectStorage:
+    # Need to enable this for artifact download links to work
```

Comment:
if this is the case, should we make the objStore route enabled by default?  (outside scope of this ticket, but if so we should open a ticket to track/follow up on)

Reply:
I don't have a strong opinion on this, this field itself is misplaced (it should be under minio), it only applies to minio which is a dev feature, so I don't think it matters that much.